app/models/direct_file_json_data.rb

```diff
@@ -17,6 +18,29 @@ class DfJsonDependent < DfJsonPerson
     json_accessor eligible_dependent: { type: :boolean, key: "eligibleDependent" }
     json_accessor is_claimed_dependent: { type: :boolean, key: "isClaimedDependent" }
     json_accessor qualifying_child: { type: :boolean, key: "qualifyingChild" }
+
+    WORDS_TO_NUMBERS = {
+      "six" => 6,
+      "seven" => 7,
+      "eight" => 8,
+      "nine" => 9,
+      "ten" => 10,
+      "eleven" => 11,
+      "twelve" => 12
+    }
```

Comment:
since not having 1-5 is a bit mystifying here, could we add a comment to clarify why those don't exist here?